README.md

```diff
@@ -75,6 +71,14 @@ The module uses some workarounds for features that are not yet available in the
 
 The module uses some workarounds for features that are not yet available in the Azure providers. Therefore, you need to have [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell) installed.
 
+### jq
+The module uses jq to extract Databricks parameters during the deployment. Therefore, you need to haved [jq](https://stedolan.github.io/jq/download/) installed.
+
+### Setting Admin Username and password
```

Comment:
Thanks for the docs about jq, the other variables are listed on the Terraform module page. I'd like to keep them in one place so that it's easier to manage.

Reply:
removed the  the hint on user credentials.